apps/browser/src/autofill/services/autofill.service.ts

```diff
@@ -1287,28 +1328,173 @@ export default class AutofillService implements AutofillServiceInterface {
    * Used when handling autofill on credit card fields. Determines whether
    * the field has an attribute that matches the given value.
    * @param {AutofillField} field
-   * @param {string} containsVal
+   * @param {string} containsValue
    * @returns {boolean}
    * @private
    */
-  private fieldAttrsContain(field: AutofillField, containsVal: string): boolean {
+  private fieldAttrsContain(field: AutofillField, containsValue: string): boolean {
     if (!field) {
       return false;
     }
 
-    let doesContain = false;
-    CreditCardAutoFillConstants.CardAttributesExtended.forEach((attr) => {
-      // eslint-disable-next-line
-      if (doesContain || !field.hasOwnProperty(attr) || !field[attr]) {
+    let doesContainValue = false;
+    CreditCardAutoFillConstants.CardAttributesExtended.forEach((attributeName) => {
+      // eslint-disable-next-line no-prototype-builtins
+      if (doesContainValue || !field.hasOwnProperty(attributeName) || !field[attributeName]) {
         return;
       }
 
-      let val = field[attr];
-      val = val.replace(/ /g, "").toLowerCase();
-      doesContain = val.indexOf(containsVal) > -1;
+      let fieldValue = field[attributeName];
+      fieldValue = fieldValue.replace(/ /g, "").toLowerCase();
+      doesContainValue = fieldValue.indexOf(containsValue) > -1;
+    });
+
+    return doesContainValue;
+  }
+
+  /**
+   * Returns a string value representation of the combined card expiration month and year values
+   * in a format matching discovered guidance within the field attributes (typically provided for users).
+   * @private
+   * @param {CardView} cardCipher
+   * @param {AutofillField} field
+   * @returns {string}
+   * @memberof AutofillService
+   */
```

Comment:
```suggestion
  /**
   * Returns a string value representation of the combined card expiration month and year values
   * in a format matching discovered guidance within the field attributes (typically provided for users).
   *
   * @param {CardView} cardCipher
   * @param {AutofillField} field
   */
```
Nit - Based on feedback I've received from other devs on the team, we want to strip out any unnecessary elements from the JSDOC that are described in the typing of the method. 

Not a a hard requirement from my side though... but just letting ya know what I've been asked to do in the past.

Reply:
love it - this is mostly generated from my Document This plugin, and I have no issue paring it down (or removing altogether; TS does the job well enough, imo)